js/index.js

```diff
@@ -12,15 +12,118 @@ const skillsList = ['C++', 'Revit', 'AutoCAD', 'Microsoft Office'];
 const skillsSection = document.getElementById('Skills');
 const skillUL = skillsSection.querySelector('ul');
 
-for (skill of skillsList){
-   let skillItem = document.createElement('li');
-   skillItem.innerHTML = skill;
-   skillItem.style.fontSize = '1.4em';
-    skillUL.appendChild(skillItem);
+for (let skill of skillsList) {
+  let skillItem = document.createElement('li');
+  skillItem.innerHTML = skill;
+  skillItem.style.fontSize = '1.4em';
+  skillUL.appendChild(skillItem);
 }
 
 document.querySelectorAll('.experience-item').forEach(item => {
-    item.addEventListener('click', () => {
-      item.classList.toggle('open');
+  item.addEventListener('click', () => {
+    item.classList.toggle('open');
+  });
+});
+
+
+//handle events for the message form
+//
+let messageForm = document.querySelector("[name='leave_message']");
+let messageSection = document.getElementById('message-section');
+let messageList = messageSection.querySelector('ul');
+messageSection.hidden = true;
+
+let idCpunter = 0;
+function makeId(){
+  let id = 'entry' + idCounter++;
+  return id;
+}
+
+let entryById={};
+
+messageForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  let name = event.target.usersName.value;
+  let email = event.target.usersEmail.value;
+  let message = event.target.usersMessage.value;
+
+  console.log('Name:', name);
+  console.log('Email:', email);
+  console.log('Message:', message);
+  let uid = makeId();
+  let newMessage = document.createElement('li');
+  newMessage.classList.add('message-item');
+
+  newMessage-innerHTML = `<a href="mailto:${email} ">${name} </a><span>wrote: ${message} </span>`;
+  newMessage.setAttribute('id', iud);
+
+  entryById[iud] = {usersName: name, usersEmail: email, usersMessage: message};
+  message.appendChild(makeEditButton());
+  newMessage.appendChild(makeRemoveButton());
+
+  messageList.appendChild(newMessage);
+  messageForm.requestFullscreen();
+  messageSection.hidden = false;
+
+});
+
+//remove button
+
+function makeRemoveButton(){
+  let makeRemoveButton = document.createElement('button');
+  removeButton.innerText = 'remove';
+  removeButton.type = 'button';
+  removeButton.className = 'remove-button';
+  removeButton.addEventListener('click', () => {
+    let entry = removeButton.parentNode;
```

Comment:
This variable name should be `removeButton`, like you're using on the lines below.

```suggestion
  let removeButton = document.createElement("button");
```